tools/kmod_scripts/sof_remove.sh

```diff
@@ -62,6 +62,7 @@ remove_module snd_soc_rt700
 remove_module snd_soc_rt711
 remove_module snd_soc_rt1308_sdw
 remove_module snd_soc_rt715
+remove_module snd_soc_cml_rt1011_rt5682
```

Comment:
no, this is in the wrong location. we group machine drivers together, then codec drivers. This should inserted line 46

Reply:
@plbossart Thank you,  Pierre. Now it is at line 46.